common/src/main/java/com/wynntils/core/webapi/profiles/item/DamageType.java

```diff
@@ -2,25 +2,20 @@
  * Copyright © Wynntils 2022.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.wc.objects.items;
+package com.wynntils.core.webapi.profiles.item;
 
-import com.google.gson.annotations.SerializedName;
+import com.wynntils.utils.StringUtils;
+import java.util.Locale;
 import java.util.regex.Pattern;
 import net.minecraft.ChatFormatting;
 
 public enum DamageType {
-    @SerializedName("NEUTRAL")
-    Neutral("❤", ChatFormatting.DARK_RED),
-    @SerializedName("EARTH")
-    Earth("✤", ChatFormatting.DARK_GREEN),
-    @SerializedName("FIRE")
-    Fire("✹", ChatFormatting.RED),
-    @SerializedName("WATER")
-    Water("❉", ChatFormatting.AQUA),
-    @SerializedName("THUNDER")
-    Thunder("✦", ChatFormatting.YELLOW),
-    @SerializedName("AIR")
-    Air("❋", ChatFormatting.WHITE);
+    NEUTRAL("✣", ChatFormatting.GOLD),
```

Comment:
Neutral changed sign and color..? Are these used for parsing, or for how we chose to present new values?

Reply:
the original values were wrong - it's only for display

Reply:
Ok!